bin/desi_qso_qn_afterburner

```diff
@@ -27,18 +29,17 @@ from quasarnp.utils import process_preds
 
 log = get_logger()
 
-## LOAD template to RE-RUN redrock
+# LOAD template to RE-RUN redrock
 qso_template_filename = [path for path in find_templates() if os.path.basename(path) == 'rrtemplate-qso.fits'][0]
 
-try: # check if the env variable QN_MODEL_FILE is defined, otherwise load boss_dr12/qn_train_coadd_indtrain_0_0_boss10.h5
-     model_QN_path = os.environ['QN_MODEL_FILE']
+try:  # check if the env variable QN_MODEL_FILE is defined, otherwise load boss_dr12/qn_train_coadd_indtrain_0_0_boss10.h5
+    model_QN_path = os.environ['QN_MODEL_FILE']
 except KeyError:
     model_QN_path = "/global/cfs/cdirs/desi/science/lya/qn_models/boss_dr12/qn_train_coadd_indtrain_0_0_boss10.h5"
```

Comment:
I will add the logger info. I don't know why but I do not have the $QN_MODEL_FILE variable set in my environment. 

I load `source /project/projectdirs/desi/software/desi_environment.sh` in my bashrc. Do you know where QN_MODEL_FILE is set ?

Reply:
$QN_MODEL_FILE is set in the desispec module file, so you should have it set immediately after `source /project/projectdirs/desi/software/desi_environment.sh`.  However, if you then run `module unload desispec` because you want to put your checkout of desispec into the path instead, the module unload process will unset that variable and you will have to set it again yourself.
```
[cori05 ~] echo $QN_MODEL_FILE

[cori05 ~] source /project/projectdirs/desi/software/desi_environment.sh
[cori05 ~] echo $QN_MODEL_FILE
/global/cfs/cdirs/desi/target/catalogs/lya/qn_models/qn_train_coadd_indtrain_0_0_boss10.h5
[cori05 ~] module unload desispec
[cori05 ~] echo $QN_MODEL_FILE

```
if that doesn't answer your situation, ping me on Slack and we'll figure it out.

Reply:
Okay ! 

Indeed, I load the desi_environment and then I unload desispec to add my version in the python path ... !